tests/global_config_s390x.py

```
@@ -0,0 +1,35 @@
+global_config = pytest_testconfig.load_python(py_file="tests/global_config.py", encoding="utf-8")
+
+utilities.constants.OS_FLAVOR_CIRROS = "fedora"
+EXPECTED_CLUSTER_INSTANCE_TYPE_LABELS[PREFERENCE_STR] = f"rhel.9.{S390X}"
+
+rhel_os_matrix = generate_os_matrix_dict(os_name="rhel", supported_operating_systems=["rhel-9-5"])
+fedora_os_matrix = generate_os_matrix_dict(os_name="fedora", supported_operating_systems=["fedora-41"])
+
+latest_rhel_os_dict, latest_fedora_os_dict = get_latest_os_dict_list(os_list=[rhel_os_matrix, fedora_os_matrix])
+
+for _dir in dir():
+    if not config:  # noqa: F821
```

Pylint (3.3.7)

[error] Using variable 'config' before assignment

(E0601)